append raster timeseries (#1183)
LayerManager/layman: CHANGELOG.md

#### Schema migrations
#### Data migrations
### Changes

> - [#1168](https://github.com/LayerManager/layman/issues/1168) Extended layer patch operations to support appending data to time-series layers.

Reviewer: ```suggestion
- [#1168](https://github.com/LayerManager/layman/issues/1168) Extend [PATCH Workspace Layer](doc/rest.md#patch-workspace-layer) with ability of appending data to existing time-series layer.
```
